graph_api/node/node_service.py

```diff
@@ -26,21 +28,22 @@ def save_node(self, node):
         Returns:
             Result of request as node object
         """
-        create_statement = "CREATE (n) RETURN n"
-        for label in node.labels:
-            create_statement = create_statement[:-10] + f":{label}" + create_statement[-10:]
+        create_template = string.Template("CREATE (n:$labels) RETURN n")
```

Comment:
string.Template seems to be more suitable for inserting values inside the string. "Guessing" the position where to put value isn't a good idea ;)